.github/workflows/flutter-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Flutter CI
+
+# This workflow is triggered on pushes to the repository.
+on: push    # Default will running for every branch.
+    
+jobs:
+  build:
+    runs-on: [windows-latest]
+    steps:
+    
+    # Setup Java environment in order to build the Android app.
+    - uses: actions/checkout@v1
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '13.x'
+    
```

Comment:
Add `on: pull_request` to run this action every time a new pull request arrives

Reply:
https://medium.com/mobile-development-group/github-actions-for-flutter-cf02923d7b5d